src/java.base/share/classes/sun/security/ssl/SSLSessionImpl.java

```diff
@@ -706,6 +723,27 @@ SecretKey getMasterSecret() {
         return masterSecret;
     }
 
+    /**
+     * Returns the exporter master secret
+     */
+    SecretKey getExporterMasterSecret() {
+        return exporterMasterSecret;
+    }
+
+    /**
+     * Returns the client's RandomCookie
+     */
+    RandomCookie getClientRandom() {
+        return clientRandom;
+    }
+
+    /**
+     * Returns the server's RandomCookie
+     */
+    RandomCookie getServerRandom() {
+        return serverRandom;
+    }
+
```

Comment:
It looks like these getters (`getExporterMasterSecret(`), `getClientRandom()`, and `getServerRandom()`) are not used yet outside of this class.